automated/linux/kselftest/kselftest.sh

```diff
@@ -107,6 +108,20 @@ while getopts "i:n:c:T:t:s:u:p:L:S:b:g:e:h" opt; do
         e)
             export ENVIRONMENT="${OPTARG}"
             ;;
+        E)
+            if [ -n "$OPTARG" ]; then
```

Comment:
This shouldn't be needed.

Reply:
This is to allow empty args, which will be the default for most tests with `-E ${ENV}` where `ENV` is not set. Without this the `-z` checks on the env_name and env_val will cause the script to exit.

Reply:
Can "$OPTARG" be "${OPTARG}" like the others please.